### ISSUE TITLE
If use the 'URLLIST' argument,reports will be only report the last url

### ISSUE THREAD

**Issue by Lz1y** [maurosoria/dirsearch#101]
If use the 'URLLIST' argument,Reports will be only report the last url.

Reports:
    --simple-report=SIMPLEOUTPUTFILE
                        Only found paths
    --plain-text-report=PLAINTEXTOUTPUTFILE
                        Found paths with status codes
    --json-report=JSONOUTPUTFILE

**Comment by maurosoria** [comment]
You are wrong, when you use a URLLIST file, dirsearch will create a file for each site in the list.

Check the AutoSave path when launching.
